libraries/kotlinx-metadata/src/kotlinx/metadata/nodes.kt

```diff
@@ -611,7 +611,7 @@ class KmValueParameter(
      * @param visitor the visitor which will visit data in this value parameter
      */
     fun accept(visitor: KmValueParameterVisitor) {
-        type?.let { visitor.visitType(it.flags)?.let(it::accept) }
+        type.let { visitor.visitType(it.flags)?.let(it::accept) }
```

Comment:
I guess the `let` call can be inlined now for simplicity
```suggestion
        visitor.visitType(type.flags)?.let(type::accept)
```